unit-test/cs_compute_tests.c

```diff
@@ -69,6 +69,22 @@ void CS_COMPUTE_TEST_CFE_TBL_ShareHandler(void *UserObj, UT_EntryKey_t FuncKey,
     *TblHandlePtr = 99;
 }
 
+void CS_COMPUTE_TEST_CFE_ES_GetModuleInfoHandler1(void *UserObj, UT_EntryKey_t FuncKey, const UT_StubContext_t *Context)
```

Comment:
Can the AppInfoHandlers be removed?

Reply:
Yes, I originally thought it was still needed for the RecomputeAppChildTask() tests, but then I realized those call ComputeApp(). I updated CS_RecomputeAppChildTask_Test_CouldNotGetAddress() to also set the return from GetLibIDByName(), so it would still not get an address.